src/steps/functions/index.ts

```diff
@@ -68,34 +65,78 @@ export async function buildCloudFunctionServiceAccountRelationships(
   );
 }
 
+export async function buildCloudFunctionSourceRepoRelationships(
+  context: IntegrationStepContext,
+): Promise<void> {
+  const { jobState } = context;
+
+  await jobState.iterateEntities(
+    {
+      _type: FunctionEntitiesSpec.CLOUD_FUNCTION._type,
+    },
+    async (cloudFunctionEntity) => {
+      const sourceRepoUrl = cloudFunctionEntity?.sourceRepoUrl as string;
```

Comment:
Can you leverage `getRawData` in order to access the property using GCP's published types?

```suggestion
      const cloudFunction = getRawData<cloudfunctions_v1.Schema$CloudFunction>(cloudFunctionEntity);
      const sourceRepoUrl = cloudFunction?.sourceRepository?.url;
```

Reply:
Nice! Wasn't aware of that generic type

Reply:
Updated in commit [488b726](https://github.com/JupiterOne/graph-google-cloud/pull/553/commits/488b72673cd5cda1c093f9854b44e667b1cfddcd) :)